doc/nrf/drivers/wifi/nrf700x/nrf700x.rst

```diff
@@ -1,36 +1,36 @@
 .. _nrf700x_wifi:
 
-nRF700X Wi-Fi driver
-####################
+nRF70 Series Wi-Fi driver
+#########################
 
 .. contents::
    :local:
    :depth: 2
 
-This driver implements the Wi-Fi® protocol for the nRF700X FullMAC series of devices.
+This driver implements the Wi-Fi® protocol for the nRF70 Series FullMAC series of devices.
 FullMAC devices implement the Wi-Fi protocol in the chipset.
 The driver configures the chipset and transfers the frames to and from the device to the networking stack.
 
-nRF700X is a companion IC and can be used with any Nordic Semiconductor System-on-Chips (SoCs) such as the nRF53 and nRF91 Series SoCs.
+nRF70 series is a companion IC and can be used with any Nordic Semiconductor System-on-Chips (SoCs) such as the nRF53 and nRF91 Series SoCs.
```

Comment:
Should this be "nRF70 series **device**".....?
```suggestion
nRF70 series is a companion IC and can be used with any Nordic Semiconductor System-on-Chips (SoCs) such as the nRF53 and nRF91 Series SoCs.
```

Reply:
Yes, I will be replacing `nRF700X` with `nRF70 Series` in throughout the NCS documentation.